lib/salus.rb

```diff
@@ -26,6 +26,7 @@ module Salus
 
   EXIT_SUCCESS = 0
   EXIT_FAILURE = 1
+  EXIT_HARD_ERROR = 2
```

Comment:
Can you provide a comment for the intended use.  This will ensure new devs can ramp up quickly on when they should leverage hard_error

Reply:
Any intent for non-configuration errors to be used here in the future?

Reply:
added comment